common/CommandControl.cpp

```diff
@@ -73,7 +73,7 @@ void LockManager::parseLockedHost(Poco::Util::LayeredConfiguration& conf)
     {
         for (size_t i = 0;; i++)
         {
-            const std::string path = "feature_lock.locked_hosts.host[" + std::to_string(i) + ']';
+            const std::string path = "feature_lock.locked_hosts.host[" + std::to_string(i) + "]";
```

Comment:
Is there any good reason for changing the single character to a string? (It is slower, that's for sure.)